tests/unit/javascript/conftest.py

```diff
@@ -25,6 +25,9 @@
 
 import pytest
 import jinja2
+from helpers.utils import CallbackChecker
+from PyQt5.QtCore import QUrl
+from qutebrowser.utils.debug import qenum_key
```

Comment:
Please also import the `utils` and `debug` modules here, and group the imports correctly.